src/config/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "anuoluwa",
+    "database": "barefoot",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
```

Comment:
Nice work so far.
Please move the db values to  a `.env` file, they contain sensitive and personal information that shouldn't be on github.

Reply:
@dewaleolaoye i would suggest you use a ```config.js``` file rather than a ```config.json``` .This would enable you to use some enviroment variable. See the screenshot for example:

![Screenshot (26)](https://user-images.githubusercontent.com/38135488/63171598-c4417500-c033-11e9-98fa-17e84f2c054a.png)

Reply:
Okay, I will implement it right away.
@zanio Thanks, that's a nice suggestion